CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 ## 4.81.0 (Unreleased)
 
+FEATURES:
+* **New Resource:** `google_biglake_catalog` ([#6152](https://github.com/hashicorp/terraform-provider-google-beta/pull/6152))
+* **New Resource:** `google_redis_cluster` ([#6158](https://github.com/hashicorp/terraform-provider-google-beta/pull/6158))
+* **New Resource:** `google_biglake_database` ([#6161](https://github.com/hashicorp/terraform-provider-google-beta/pull/6161))
+* **New Resource:** `google_compute_network_attachment` ([#6159](https://github.com/hashicorp/terraform-provider-google-beta/pull/6159))
+
+IMPROVEMENTS:
+* compute: made the field `distribution_policy_target_shape` of `google_compute_region_instance_group_manager` not cause recreation of the resource. ([#6156](https://github.com/hashicorp/terraform-provider-google-beta/pull/6156))
```

Comment:
```suggestion
* compute: allowed the field `distribution_policy_target_shape` of `google_compute_region_instance_group_manager` to be updatable. ([#6156](https://github.com/hashicorp/terraform-provider-google-beta/pull/6156))
```

Reply:
It was suggested on the TPG release note to make this note focusing on the user impact. Thanks for the suggestion here.